ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsInitializeIngestion.scala

```diff
@@ -19,6 +20,10 @@ private[backend] trait StorageBackendTestsInitializeIngestion
 
   import StorageBackendTestValues._
 
+  val now = Timestamp.now()
```

Comment:
Please move it to `StorageBackendTestValues`

Reply:
Repalaced with `someTime`